### PR TITLE
Added prototype comparison page

### PR DESCRIPTION
**Pull request by SethCKL**
Abstracted existing search location functionality into a PlaceSearch component for repeated use

Implemented a MantineTable component using Mantine React Table also for repeated use

### CODE REVIEW

File: src/components/ui/MantineTable.tsx
Comment (zhekaiii) [inline src/components/ui/MantineTable.tsx:1]: Do you think we can name this a little bit more specific? Like `PlaceComparisonTable`?

File: src/components/ui/MantineTable.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    optionB: '420'
+  },
+  {
+    optionA: '69',
+    delta: 'B is better',
+    optionB: '420'
+  },
+];
+
+export default function App() {
```
Comment (zhekaiii) [inline src/components/ui/MantineTable.tsx:29]: Let's stick to our usual convention of declaring Functional Components, and also please name the component properly :)
```suggestion
export const PlaceComparisonTable: React.FC = () => {
```

File: src/components/ui/MantineTable.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { useMemo } from 'react';
+import {
+  MantineReactTable,
+  useMantineReactTable,
+  type MRT_ColumnDef, //if using TypeScript (optional, but recommended)
+} from 'mantine-react-table';
+
+//If using TypeScript, define the shape of your data (optional, but recommended)
+interface Compare {
```
Comment (zhekaiii) [inline src/components/ui/MantineTable.tsx:9]: Compare is a verb, not the most suitable for naming after types. Also, sorry for another unspoken rule/convention but let's use typescript `types` over `interfaces`

**Review by zhekaiii — CHANGES_REQUESTED** [review]
Hi, I know this is just an early prototype but here are my comments :)

**Comment by zhekaiii** [comment]
image
Can you also find some way to hide this? Or we do in a future PR when we put actual data in?

**Comment by SethCKL** [comment]
Yes actually! but i couldn't figure out how to only make it display only when two places have been selected

**Review by zhekaiii — APPROVED** [review]
LGTM, thanks!

**Pull request merged**